ci/images/ci-runner/hack/bin/deploy-cluster.sh

```diff
@@ -14,58 +14,93 @@ source "$SCRIPT_DIR/utils.sh"
 fetch_bitwarden_secrets() {
     printf "Fetch secrets from bitwarden server\n" | indent 2
     open_bitwarden_session
-    get_base_domain
-    get_pull_secret
     get_aws_credentials
+    get_rosa_token
 }
 
 print_debug_info() {
     printf "Print debug info......\n" | indent 2
+    rosa describe cluster --cluster="$CLUSTER_NAME"
+}
 
-    printf "HostedCluster CR Status: " | indent 4
-    kubectl -n clusters get hostedcluster "$CLUSTER_NAME" -o yaml
-
-    printf "Control Plane pods of HostedCluster: " | indent 4
-    kubectl -n "clusters-${CLUSTER_NAME}" get pods
+# Add a function to check all of clusteroperators till AVAILABLE are true
```

Comment:
`# Add a function to check all of clusteroperators are AVAILABLE`

Reply:
fixed